draft-ietf-httpbis-expect-ct.md

```diff
@@ -274,6 +274,17 @@ delta-seconds = 1*DIGIT
 
 `delta-seconds` is used as defined in Section 1.2.1 of RFC 7234 {{!RFC7234}}.
 
+### Examples
+
+The following examples demonstrate valid Expect-CT response header fields:
+
+~~~ inline
+Expect-CT: max-age=86400,enforce
+Expect-CT: max-age=86400, enforce, report-uri="https://foo.test/report"
+Expect-CT: max-age=86400,report-uri="https://foo.test/report"
```

Comment:
I would put a blank line between each example, lest this be confused for a single header block.

Also, use foo.example, not foo.test.